qa/rpc-tests/segwit.py

```diff
@@ -211,7 +211,25 @@ def run_test(self):
         block = self.nodes[2].generate(1) #block 432 (first block with new rules; 432 = 144 * 3)
         sync_blocks(self.nodes)
         assert_equal(len(self.nodes[2].getrawmempool()), 0)
-        assert_equal(len(self.nodes[2].getblock(block[0])["tx"]), 5)
+        segwit_tx_list = self.nodes[2].getblock(block[0])["tx"]
+        assert_equal(len(segwit_tx_list), 5)
+
+        print("Verify block and transaction serialization rpcs return differing serializations depending on rpc serialization flag")
+        # Note: node1 has version 2, which is simply >0 and will catch future upgrades in tests
+        assert(self.nodes[2].getblock(block[0], False) !=  self.nodes[0].getblock(block[0], False))
+        assert(self.nodes[1].getblock(block[0], False) ==  self.nodes[2].getblock(block[0], False))
+        for i in range(len(segwit_tx_list)):
+            #Coinbase can not be a segwit transcation
```

Comment:
The coinbase case is when serialization should match regardless of segwit/non-segwit combos as there is no witness.  

When I say "segwit transaction"(oops, typo) I mean to say "no witness data". Will clarify.